src/pages/Home/index.js

```diff
@@ -55,12 +55,12 @@ function Home() {
       </div>
       {!!toggleFilter && <Filter films={films} setFilms={setFilms} />}
       <div className="film-cards-container">
-        {films.map((item, key) => (
-              <FilmCard 
-                key={key}
-                film={item} callFilm={callFilm}/>
+        {films.map((film, key) => (
+            <React.Fragment key={key}>
```

Comment:
I'm not sure we need a react fragment in this line

Reply:
it's the same as <> </>, but I like more this way, you had put another <div> to contain  "film-cards-container" that contains a "film-card-container", the difference between that <div> you had put and the <></>(fragment) is that this, in HTML doesn't actually have a real tag :v, so that the Liliana's CSS are not affected by that extra container(without any real functionality), the "function" of "clicking" I added it to the same card-content/image, so that it works independently (and with a better hitbox, before, it has almost 50-70px more because of the container, so you could click outside the "films-card-container" and still get redirected to the page without any sense), and the Link is outside of the <> because we don't need 24Links with the same function :v